packages/core/src/api/plugin/Plugin.tsx

```diff
@@ -78,6 +94,19 @@ export default class Plugin {
           outputs.push({ type: 'widget', widget });
         },
       },
+      featureFlags: {
+        registerFeatureFlag(name) {
```

Comment:
plugins need to be able to read feature flags during registration too

Reply:
Fixed.